Podcast/EpisodeDetailViewController.swift

```diff
@@ -8,51 +8,99 @@
 
 import UIKit
 
-class EpisodeDetailViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
-    var scrollView: UIScrollView!
-    var episodeDetailHeaderView: EpisodeDetailHeaderView!
+class EpisodeDetailViewController: UITableViewController, EpisodeDetailHeaderViewCellDelegate {
     var episode: Episode?
-    var commentsTableViewHeader: CommentsTableViewHeader!
-    var commentsTableView: UITableView!
     var comments: [Comment] = Array.init(repeating: Comment(episodeId: "", creator: "Mark Bryan", text: "Great point here!!", creationDate: "5 months ago", time: "5:13"), count: 10)
+    var episodeDetailViewHeaderHeight: CGFloat?
```

Comment:
because it's set at run time once the cell has setup its subviews. i could also just set it to 0 and overwrite that value once the height is determined. that's probably better